rafthttp/peer.go

```diff
@@ -169,9 +170,6 @@ func startPeer(tr http.RoundTripper, urls types.URLs, local, to, cid types.ID, r
 						m.Type, p.id, name, bufSizeMap[name])
 				}
 			case mm := <-p.recvc:
-				if mm.Type == raftpb.MsgApp {
```

Comment:
It is explained here: https://github.com/yichengq/etcd/commit/eb3db691b76625a434ad86732999f16fd61aaa9f